docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Good point.

Another approach could be an [enscope-like](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/contrib/enscope) command-line flag to specify the labels that should be used on all objects created.